sriovnet_helper.go

```diff
@@ -9,6 +9,7 @@ import (
 
 const (
 	NetSysDir        = "/sys/class/net"
+	PciSysDir        = "/sys/bus/pci/devices/"
```

Comment:
no trailing /

Reply:
sorry it not belong to this patch I will remove it

Reply:
ok